pkg/apis/hub/v1alpha1/api_access.go

```diff
@@ -47,9 +45,9 @@ type APIAccessSpec struct {
 	// +optional
 	Groups []string `json:"groups"`
 
-	// AnyGroups states that everyone will gain access to the selected APIs.
+	// Everyone states that everyone will gain access to the selected APIs.
 	// +optional
-	AnyGroups bool `json:"anyGroups"`
+	Everyone bool `json:"everyone"`
 
 	// APISelector selects the APIs that will be accessible to the configured user groups.
```

Comment:
The `APICollectionSelector` and `APICollections` field are related struct can be removed.